crates/catalog-unity/src/lib.rs

```diff
@@ -616,10 +631,11 @@ impl UnityCatalog {
                 table_name.as_ref()
             ))
             .header(AUTHORIZATION, token)
-            .send_retry(&self.retry_config)
-            .await?;
+            .send()
+            .await
+            .map_err(UnityCatalogError::from)?;
 
-        Ok(resp.json().await?)
+        Ok(resp.json().await.map_err(UnityCatalogError::from)?)
```

Comment:
if we have an `impl From<...>` for the error, why do we need the map? does the function return a different kind of err.

Reply:
This is a bit of jank, the base error type in core no longer has a from for rewqest::Error so we convert to our own error type that does have a From to DataCatalogError